### ISSUE TITLE
Default to '-p Solid' for OBJ and OFF

### ISSUE THREAD

**Issue by balazsdukai** [tudelft3d/val3dity#72]
Consider the test `test_completely_empty` on v2_test branch. It reads files that are completely empty, only their extension differ, such as:

```
                params=["completely_empty.gml",
                        "completely_empty.xml",
                        "completely_empty.obj",
                        "completely_empty.off",
                        "completely_empty.json",])
```

I would expect that running val3dity with the same command for each file eg `$ val3dity completely_empty.obj` returns the same error for all of them, *901 - Invalid input file*. However, for the OBJ and OFF files the error 903 is returned, because of the missing `-p Solid/MultiSurface` argument in the call I think.

If each accepted input format can hold geometrically the same city model, then why does the user need to run val3dity differently for OBJ/OFF and CityGML/CityJSON? The use of the `-p` argument is clear from the documentation, but would it make sense that it defaults to `-p Solid` for each input format? In that case the same command could be used to validate a city model, regardless of the file format.

**Comment by hugoledoux** [comment]
okay fixed, default to Solid if nothing passed for POLY/OBJ/OFF

**Comment by balazsdukai** [comment]
I've compiled after b1a0e0b and it looks like that -p defaults to MultiSurface, furthermore the `-p Solid` option is not respected for obj/off/poly.

See the tests on the current v2_test, `pytest -k "options_primitive" -v`

**Comment by hugoledoux** [comment]
? You forgot to pull surely, here all good. I had triple-checked and did it again

**Comment by balazsdukai** [comment]
Fixed by d6d31b4d5de10d7a5755658a955b0f857a9f0e09
